cache_path_test.go

```diff
@@ -221,14 +223,14 @@ func Test_parseIgnoreList(t *testing.T) {
 	}
 }
 
-func createDirStruct(t *testing.T, pths []string) {
-	for _, pth := range pths {
+func createDirStruct(t *testing.T, contentByPth map[string]string) {
+	for pth, content := range contentByPth {
 		dir := filepath.Dir(pth)
 		if err := os.MkdirAll(dir, 0777); err != nil {
 			t.Fatalf("failed to create dir: %s", err)
 			return
 		}
-		if _, err := os.Create(pth); err != nil {
+		if err := fileutil.WriteStringToFile(pth, content); err != nil {
```

Comment:
You changed the method, but the error message is the same.
Could you update the error message as well in your next PR?

Reply:
good catch, thx